src/Tokenizer/Tokens.php

```diff
@@ -705,7 +705,7 @@ public function getPrevMeaningfulToken($index)
     /**
      * Find a sequence of meaningful tokens and returns the array of their locations.
      *
-     * @param array      $sequence      an array of tokens (same format used by getNextTokenOfKind)
+     * @param array      $sequence      an array of tokens (kinds) (same format used by getNextTokenOfKind)
```

Comment:
because you can pass an array of Tokens or an array of tokens kinds;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.1/src/Tokenizer/Tokens.php#L734